DOCSP-14304: document directConnection (#145)
mongodb/docs-node: source/fundamentals/connection.txt

   mongodb://host1:27017,host2:27017,host3:27017/?replicaSet=myRs

.. note::

>    Although it is possible to connect to a replica set deployment by
>    providing only a single host, you should provide the driver with the
>    full list to ensure that it is able to connect in the event the specified host is unavailable.

Reviewer: **suggestion:** Tighten the language somewhat using second person and imperative mood.
```suggestion
   You only need to provide a single host to connect to a deploy replica set deployment. 
   To ensure connectivity when the specified host is unavailable, provide the driver with
   the full list of hosts
```

Direct Connection
+++++++++++++++++

> By default, the driver will discover all members of the replica set given the address of any one member,
> and will dispatch operations to the appropriate member, such as writes against the **primary**.

Reviewer: **suggestion:** Change to present tense and split into a bulleted list
```suggestion
When making a direct connection, the driver takes the following actions by default:

- Discovers all replica set members when given the address of any one member.
- Dispatches operations to the appropriate member, such as writes against the **primary**.
```

> To force operations on the server designated in the connection URI, specify the ``directConnection`` option.

Reviewer: **question:** Should this be `host` to be consistent?
```suggestion
To force operations on the host designated in the connection URI, specify the ``directConnection`` option.
```

   mongodb://host1:27017,host2:27017,host3:27017/?replicaSet=myRs

When making a connection, the driver takes the following actions by default:

- Discovers all replica set members when given the address of any one member.
- Dispatches operations to the appropriate member, such as write against the **primary**.

.. tip::

   You only need to specify one host to connect to a replica set. However, to ensure connectivity if the specified host
   is unavailable, provide the full list of hosts.

Direct Connection
+++++++++++++++++

To force operations on the host designated in the connection URI, specify the ``directConnection`` option. Direct
connections:

> - do not support SRV strings.
> - will error on writes when the specified host is not the **primary**.
> - require :manual:`enabling secondary reads </reference/method/rs.secondaryOk/>` when the specified host in not the
>   **primary**.

Reviewer: **suggestion:** Clean a little of the language.

```suggestion
- Don't support SRV strings.
- Fails on writes when the specified host is not the **primary**.
- Require :manual:`enabling secondary reads </reference/method/rs.secondaryOk/>`
   when the specified host isn't the **primary**.
```

       server documentation on the
       :manual:`w Option </reference/write-concern/#w-option>`.

   * - **directConnection**
     - boolean
     - ``false``

>      - Specifies whether to force dispatch **all** operations to the server

Reviewer: **questions:**

1. Would it make more sense to say "funnels all operations"? This makes it clear it could create a bottleneck or other undesirable condition.
2. Would it not be `host`?

Author: Definitely agree with host. Funnel is incorrect here, it actually dispatches to the host.
